### DISCUSSION TITLE
Group key replacement

### DISCUSSION THREAD

**Discussion by IlyaRazuvaev** [taskforcesh/bullmq#1645] [Q&A] [answered]
Hi there.
Group key were removed since Bull 3.0. Is there any community approved way to work with rate limiters per customer now?

**Comment by manast** [comment, marked as answer]
You can use Bull instead https://github.com/optimalbits/bull 
But let me explain that the group's key feature was removed for a reason, the implementation is not so great, although it seems to do its work for many users.
